pyEpiabm/pyEpiabm/tests/test_place.py

```diff
@@ -0,0 +1,70 @@
+import unittest
+import pyEpiabm as pe
+
+
+class TestPlace(unittest.TestCase):
+    """Test the place class.
+    """
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.pop = pe.Population()
+        cls.pop.add_cells(1)
+        cls.cell = cls.pop.cells[0]
+        cls.pop.cells[0].add_microcells(1)
+        cls.microcell = cls.cell.microcells[0]
+        cls.pop.cells[0].microcells[0].add_people(1)
+        cls.person = cls.pop.cells[0].microcells[0].persons[0]
+
+    def test_construct(self):
+        """Tests constructor method.
+        """
+        new_cell = pe.Cell()
+        self.assertRaises(KeyError, pe.Place, (1, 1), pe.PlaceType.Hotel,
+                          new_cell, self.microcell)
+        test_place = pe.Place((1, 1), pe.PlaceType.Hotel, self.cell,
+                              self.microcell)
+        self.assertEqual(len(test_place.persons), 0)
+
+    def test_change_persons(self):
+        """Tests the add and remove persons functions.
+        """
+        test_place = pe.Place((1, 1), pe.PlaceType.Hotel, self.cell,
+                              self.microcell)
+        test_place.add_person(self.person)
+        self.assertEqual(len(test_place.persons), 1)
+
+        test_place.remove_person(self.person)
+        self.assertEqual(len(test_place.persons), 0)
+        self.assertRaises(KeyError, test_place.remove_person, self.person)
+
+        test_place.add_person(self.person)
+        test_place.add_person(pe.Person(self.microcell))
+        self.assertEqual(len(test_place.persons), 2)
+        test_place.empty_place()
+        print(test_place.persons)
+        self.assertEqual(len(test_place.persons), 0)
+
+    def test_set_susc(self):
+        test_place = pe.Place((1, 1), pe.PlaceType.Hotel, self.cell,
+                              self.microcell)
+        self.assertEqual(test_place.susceptibility, 0)
+        test_place.set_susceptibility(10)
```

Comment:
Again, these should strictly be floats, but as we're not testing that here it's not the end of the world